reference/config_files/global_conf.rst

```diff
@@ -36,6 +36,37 @@ have priority over globally defined ones in *global.conf*, and can be defined as
     tools.build:jobs=10
 
 
+Configuration file template
+++++++++++++++++++++++++++++
+
+Available since: `1.46.0 <https://github.com/conan-io/conan/releases>`_
+
+It is possible to use **jinja2** template engine for *global.conf*. When Conan loads this file, immediately parses
+and renders the template, which must result in a standard tools-configuration text.
+
+  .. code:: jinja
+
+     [conf]
+     # Using all the cores automatically
```

Comment:
```suggestion
     # Using all the cores automatically
```

global.conf doesn't have the [conf] section, everything is conf

Reply:
Then, it's wrong in the `Tools configurations` section above as well.